last_refresh.php

```diff
@@ -37,7 +38,10 @@
 $form = new \local_envbar\form\lastrefresh(null, array('prodlastcheck' => $config->prodlastcheck));
 
 if ($data = $form->get_data()) {
+    // Update the prodlastcheck and clear the cache to make it effective.
     set_config('prodlastcheck', $data->lastrefresh, 'local_envbar');
+    $cache = cache::make('local_envbar', 'records');
```

Comment:
Can you please refactor this logic to here:

https://github.com/catalyst/moodle-local_envbar/blob/master/classes/local/envbarlib.php#L402

so we only have one touch point